build/config.js

```diff
@@ -57,6 +64,8 @@ const genConfig = (name) => {
         input: opts.entry,
         external: opts.external,
         output: {
+            strict: false,
```

Comment:
@farfromrefug Can you explain why this is required? 

According to rollup docs the `strict` option does the following:
> Whether to include the 'use strict' pragma at the top of generated non-ESM bundles. Strictly-speaking, ES modules are always in strict mode, so **you shouldn't disable this without good reason**.

Reply:
I had some issues with Vue and Nativescript in strict mode and {N} guys told me to remove strict. I did it while i was testing. Will remove it